### ISSUE TITLE
Input of variable length

### ISSUE THREAD

**Issue by mireklzicar** [jrzaurin/pytorch-widedeep#151]
I want to train a deep tabular model, but I don't have a nice DataFrame of fixed dimensions – every row has different number of features. With tabular transformer models it should be still possible to train it with varying-length input, right? Is there some recommended way in the library to do this easily? Any hints?

**Comment by jrzaurin** [comment]
Hey there @mireklzicar. I am traveling right now Ill reply asap!

**Comment by jrzaurin** [comment]
Hey @mireklzicar 

mmm..🤔, the library, when working with tabular data, is expected to have all rows with the same number of columns. From the top of my head I cannot think of an easy way of implemented diff length rows in our library. 

I could think of some type of dynamic padding with a custom dataloader or via a collate function. Give me a few days and let me come back to you. If you have any idea, write it here and I can give it a go

And thanks for opening the issue!
